src/api/java/com/minecolonies/api/colony/managers/interfaces/ICitizenManager.java

```diff
@@ -60,6 +61,8 @@ default ICitizenData spawnOrCreateCitizen(final ICitizenData data, @NotNull fina
     @Override
     ICitizenData createAndRegisterCivilianData();
 
```

Comment:
javadoc

Reply:
Do you mean I need to add the /** desription of the method **/ ?

Reply:
yes